### PR TITLE
Update Travis config

### PR DESCRIPTION
**Pull request by GaryJones**
* Allow HHVM to fail.
* Put PHPCS and WPCS into `/tmp/` subdirectories so they don't have to be excluded from linting.
* Use 2.3.0 tag of PHPCS, since master (currently 2.3.1) includes a call to `Phar` class.
* Specify master branch of WPCS, which is more stable than default (develop) branch.
* Simplify linting command.
* Clarify WPCS standard.
* Add summary report for PHPCS.

Tested against a fork of this repo, since this repo is not yet hooked up to Travis.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,40 +10,41 @@ language: php
...
     # Install CodeSniffer for WordPress Coding Standards checks.
-    - git clone https://github.com/squizlabs/PHP_CodeSniffer.git phpcs
+    # 2.3.1 breaks PHP 5.2 on Travis due to use of Phar class.
+    - git clone -b 2.3.0 --depth 1 https://github.com/squizlabs/PHP_CodeSniffer.git $PHPCS_DIR
     # Install WordPress Coding Standards.
-    - git clone -b master https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git wpcs
-    # Hop into CodeSniffer directory.
-    - cd phpcs
+    - git clone -b master --depth 1 https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards.git $WPCS_DIR
     # Set install path for WordPress Coding Standards.
```
Comment (jrfnl) [inline .travis.yml]: +1 on limiting the clone depth!

**Comment by shivapoudel** [comment]
@GaryJones If possible Utilize the scrutinizer too ;)

**Comment by GaryJones** [comment]
@shivapoudel #330 ;-)

**Comment by shivapoudel** [comment]
Cool!

**Pull request merged**